lib/boxxspring/resources/advertising_advertisement.rb

```diff
@@ -0,0 +1,17 @@
+module Boxxspring
+
+  class AdvertisingAdvertisement < Base
+    field  :id
+    field  :property_id
+    field  :name
+    field  :created_at
+    field  :updated_at
```

Comment:
Remove state field on all the ad models bc it doesn't exist. Add fields id, property_id, created_at, updated_at.